index.js

```diff
@@ -378,11 +378,13 @@ function Argv (processArgs, cwd) {
   }
 
   var versionOpt = null
-  self.version = function (ver, opt, msg) {
-    versionOpt = opt || 'version'
-    usage.version(ver)
+  self.version = function (opt, msg, ver) {
+    var version = ver || msg || opt || undefined
```

Comment:
I always just want to customize the option (usually `-v` with `--version` alias) and it's annoying to `require(../package.json).version` every time. (Though we probably shouldn't use `require()` internally for this, to avoid the dynamic require problem.)